Shared/dugga.js

```diff
@@ -1164,13 +1164,13 @@ function AJAXService(opt,apara,kind)
 				serviceURL= "sectionedservice.php"; //change when MS is created
 				break;
 			case "CREGITEX":
-				serviceURL= "../DuggaSys/microservices/sectionedService/changeActiveCourseVersion_sectioned_ms.php";
+				serviceURL= "../DuggaSys/microservices/sectionedService/updateActiveCourseVersion_sectioned_ms.php";
```

Comment:
Seems like the original microservice url was wrong here. It should be using createGithubCodeExample_ms.php